No-Ones-Jupyter-Notebook.ipynb

```diff
@@ -1,15 +1,27 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "This notebook contains Julia code used to generate results which will be published in the `Numbers Without Ones (Revisited)` article in the `Recreational Mathematics Magazine`.\n",
+    "\n",
+    "## Level Finding\n",
+    "\n",
+    "An integer is ``one-free'' if none of its digits is 1. The level of a positive integer n is the smallest positive integer m such that n*m is one-free. Let's start by defining a function to calculate the level of any given n using a brute force approach that terminates when the level has been found or when all values up to some maximum value have been tested."
+   ]
+  },
```

Comment:
How about this? "An integer is *one-free* if none of its digits is 1.  The *level* of a positive integer n is the smallest positive integer m such that n*m is one-free.  The following cell defines a function that (by brute force) calculates the level of m if that level is less than some maximum value. If the level of n is undefined or exceeds the maximum value the function returns  ``error''.
(Note:  I am following the convention that words being defined are in italics rather than quotes.)

Reply:
For this one I tried to make the change using the github UI.  See the "Add a suggestion" button directly above the text box when you are leaving a comment?  That's what I used.  See the comment below on the line of code (text) that you'd like to change.  I think you can press "Commit suggestion" and it'll automatically change the code.  I don't know if the notebook will reflect the change right away.  Notebooks are a little wonky.